js/app.js

```diff
@@ -92,32 +54,79 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
             };
             else
                 params = {};
-            params.finish_order = $scope.address;
+            params.finish_order = $scope.order_id;
             url = window.location.pathname;
             var serializedParams = $httpParamSerializer(params);
             if (serializedParams.length > 0) {
                 url += ((url.indexOf('?') === -1) ? '?' : '&') + serializedParams;
             }
             return url;
         }
-
-    //Create url for altcoin payment
-    $scope.alt_track_url = function(uuid) {
-        var params = getParameterByNameBlocko('wc-api');
-        if (params)
-            params = {
-                "wc-api": params
-            };
-            else
-                params = {};
-            params.uuid = uuid;
-            url = window.location.pathname;
-            var serializedParams = $httpParamSerializer(params);
-            if (serializedParams.length > 0) {
-                url += ((url.indexOf('?') === -1) ? '?' : '&') + serializedParams;
-            }
-            return url;
+    
+    // Select Blockonomics currency
+    $scope.select_blockonomics_currency = function(blockonomics_currency) {
+        $scope.currency = blockonomics_currency;
+        $scope.currency_selecter  = false;
+        $scope.spinner = true;
+        //Check if the bitcoin address is present
+        if (typeof $scope.order_id != 'undefined') {
+            //Fetch the order using address
+            Order.get({
+                "get_order": $scope.order_id,
+                'crypto': blockonomics_currency
+            }, function(data) {
+                $scope.order = data;
+                //Check the status of the order
+                if ($scope.order.status == -1) {
+                  $scope.clock = $scope.order.timestamp + totalTime - Math.floor(Date.now() / 1000);
+                    //Mark order as expired if we ran out of time
+                    if ($scope.clock < 0) {
+                    $scope.order.status = -3;
```

Comment:
Minor thing but as JS tends to get messy its good practice to intend correctly, same goes for other lines